### PR TITLE
sig-auth chair change

### PR DESCRIPTION
**Pull request by liggitt**
As discussed on the [mailing list](https://groups.google.com/d/msg/kubernetes-sig-auth/kqhoYhrzYLc/Bf4YLnFJAQAJ) and in the [20181003 sig-auth meeting](https://www.youtube.com/watch?v=9EA16cWHY7g&t=100).

Opening for proposal and holding for lazy consensus per [governance roles doc](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#notes-on-roles).

/hold
/assign @mikedanese @tallclair 
cc @kubernetes/sig-auth-misc @enj

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -10,8 +10,8 @@ aliases:
     - bgrant0607
     - jdumars
   sig-auth-leads:
-    - liggitt
     - mikedanese
+    - eng
```
Comment (enj) [inline OWNERS_ALIASES]: typo, should be `enj` :smile:

**Comment by cblecker** [comment]
@liggitt are you retaining your tech lead role? this is what I can see from the e-mail list.. however this PR moves you into the emeritus section

**Comment by liggitt** [comment]
> @liggitt are you retaining your tech lead role? this is what I can see from the e-mail list.. however this PR moves you into the emeritus section

I am. Until I opened this PR, I didn't realize there was a mechanism for listing them in sigs.yaml, and didn't realize we hadn't yet enumerated ours for sig-auth.

I [started a thread](https://groups.google.com/forum/#!topic/kubernetes-sig-auth/v0-Awf0rBOg) about getting that list defined in a more methodical fashion.

**Comment by tallclair** [comment]
Looks good. I propose we leave this open until next Friday, 2018-10-12.

**Comment by cblecker** [comment]
yeah, they can be listed in a section like below:
https://github.com/kubernetes/community/blob/f820675bd0f81694b58dbf377906b95fdc15563c/sigs.yaml#L823-L828

If there is anything I can do to help, please let me know :)

**Comment by mikedanese** [comment]
LGTM

**Comment by tallclair** [comment]
As our proposed hold date (2018-10-12) has passed, I propose we merge this as is, and follow up with the techleads piece. WDYT?

/lgtm

**Comment by cblecker** [comment]
/approve
Will leave removing the hold to sig-auth leads when ready.

**Pull request merged**